docs/sections/pipeline_samples/tutorials/generate_preference_dataset.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.        load_dataset &gt;&gt; generate_responses &gt;&gt; group_responses &gt;&gt; evaluate_responses &gt;&gt; [format_dpo, to_argilla]
</pre><p>again, should we do them sequential, or do you want to show to [] opearation along with the &gt;&gt;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>In this case, I prefer to show [] to show the difference, as Format already shows the chosen/rejected automatically, but with argilla is manually</p>